app/src/main/java/com/example/trainingintive/navigators/MainNavigator.kt

```diff
@@ -22,10 +22,9 @@ class MainNavigator @Inject constructor() : Navigator() {
     }
 
     private fun logoutAndDisplayStartScreen() {
-        // TODO You are using let but not using it's argument (it) so it is redundant, probably it is some typo and it should be used like in `showSnackBar`
-        activity?.let {
-            AuthUI.getInstance().signOut(it) // TODO It should be called from data layer (repository) and you should call it from ViewModel via UseCase
```

Comment:
You should leave this TODO because it isn't resolved :) You don't need to do this now, but leave this TODO to remember about it.